bundler/lib/dependabot/bundler/helpers.rb

```diff
@@ -4,7 +4,7 @@ module Dependabot
   module Bundler
     module Helpers
       V1 = "1.17.3"
-      V2 = "2.3.8"
+      V2 = "2.3.9"
```

Comment:
Then Bundler version selection behavior implemented in this file should be the default as long as there's single v1 and v2 versions installed in the running environment. If that's the case I think it should be possible to completely remove this logic (don't pass `BUNDLER_VERSION` at all to native helpers) and let RubyGems do the right thing. But it seems something to consider for the future, definitely not for this PR.

Reply:
Aye, yeah we should definitely try that and see if it doesn't break anything!